src/lib.rs

```diff
@@ -40,6 +40,7 @@ mod preprocessor;
 mod pretty_printer;
 pub(crate) mod printer;
 pub mod style;
+mod syntax_analysis;
```

Comment:
Bikeshedding, but maybe use another name here? It sounds a bit like we are actually performing syntax analysis (like on an AST after parsing) here. Maybe `syntax_set_analysis`? Or `syntax_dependencies`.

Reply:
Let's go with `syntax_dependencies` :+1: